src/Workspaces/Core/Portable/Shared/Extensions/INamedTypeSymbolExtensions.cs

```diff
@@ -419,12 +419,101 @@ private static ImmutableArray<ISymbol> GetMembers(INamedTypeSymbol type, ISymbol
         public static INamespaceOrTypeSymbol GenerateRootNamespaceOrType(this INamedTypeSymbol namedType, string[] containers)
         {
             INamespaceOrTypeSymbol currentSymbol = namedType;
-            for (int i = containers.Length - 1; i >= 0; i--)
+
+            for (var i = containers.Length - 1; i >= 0; i--)
             {
                 currentSymbol = CodeGenerationSymbolFactory.CreateNamespaceSymbol(containers[i], members: new[] { currentSymbol });
             }
 
             return currentSymbol;
         }
+
+        public static ImmutableArray<ISymbol> GetOverridableMembers(
+            this INamedTypeSymbol containingType, CancellationToken cancellationToken)
+        {
+            var result = new Dictionary<ISymbol, int>();
+            var index = 0;
```

Comment:
Fair enough.  I'll comment.  But hte general idea was that we walk from the topmost type down to whatever type you're in.  I wanted  to preserve ordering of overrides as we see them.  So 'object' overrides will be at the top, then the next closest types' overriable members, and so on.  This approach allows us to do that.